Doc/tools/templates/layout.html

```diff
@@ -61,7 +61,13 @@
    // since all the "active, built and not hidden" versions will be shown automatically.
    let versions = config.versions.active.concat([
        {
-           slug: "dev (3.13)",
+           slug: "dev (3.14)",
+           urls: {
+               documentation: "https://docs.python.org/3.14/",
+           }
+       },
+       {
+           slug: "3.13",
```

Comment:
It's currently still in pre-release:

```suggestion
           slug: "pre (3.13)",
```